components/english/english-speaking.tsx

```
@@ -0,0 +1,162 @@
+"use client";
+
+import { useState } from "react";
+import SpeechRecognition, { useSpeechRecognition } from "react-speech-recognition";
+import { cn } from "@/lib/utils";
+import { ArrowRight, Mic, RefreshCw, X, Check, Volume2 } from "lucide-react";
```

ESLint

[error] 'RefreshCw' is defined but never used.

(@typescript-eslint/no-unused-vars)